net/haproxy/src/etc/rc.syshook.d/50-haproxy.stop

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+. /etc/rc.conf.d/haproxy
+
+if [ "x${haproxy_hardstop}" == "xYES" ]; then
```

Comment:
we have to hard stop here unconditionally, this is the reboot, otherwise still stuck